src/services/codefixes/importFixes.ts

```diff
@@ -250,8 +275,69 @@ function createImportAdderWorker(sourceFile: SourceFile, program: Program, useAu
         const useRequire = shouldUseRequire(sourceFile, program);
         const fix = getImportFixForSymbol(sourceFile, Debug.checkDefined(exportInfo), program, /*position*/ undefined, !!isValidTypeOnlyUseSite, useRequire, host, preferences);
         if (fix) {
-            addImport({ fix, symbolName, errorIdentifierText: undefined });
+            addImport({ fix, symbolName: localName ?? symbolName, errorIdentifierText: undefined });
+        }
+    }
+
+    function addImportForNonExistentExport(exportName: string, exportingFileName: string, exportKind: ExportKind, exportedMeanings: SymbolFlags, isImportUsageValidAsTypeOnly: boolean) {
+        const exportingSourceFile = program.getSourceFile(exportingFileName);
+        const useRequire = shouldUseRequire(sourceFile, program);
+        if (exportingSourceFile && exportingSourceFile.symbol) {
+            const { fixes } = getImportFixes(
+                [{
+                    exportKind,
+                    isFromPackageJson: false,
+                    moduleFileName: exportingFileName,
+                    moduleSymbol: exportingSourceFile.symbol,
+                    targetFlags: exportedMeanings,
+                    symbol: undefined!,

```

Comment:
It would be good if there was a less sketchy way to do this; I'll look a little closer